kedro/contrib/io/azure/csv_blob.py

```diff
@@ -72,18 +74,20 @@ def _describe(self) -> Dict[str, Any]:
             blob_from_text_args=self._blob_from_text_args,
             load_args=self._load_args,
             save_args=self._save_args,
+            version=self._version,
         )
 
     # pylint: disable=too-many-arguments
     def __init__(
         self,
         filepath: str,
         container_name: str,
-        credentials: Dict[str, Any],
+        credentials: Optional[Dict[str, Any]] = None,
         blob_to_text_args: Optional[Dict[str, Any]] = None,
         blob_from_text_args: Optional[Dict[str, Any]] = None,
         load_args: Optional[Dict[str, Any]] = None,
         save_args: Optional[Dict[str, Any]] = None,
+        version: Version = None,
     ) -> None:
         """Creates a new instance of ``CSVBlobDataSet`` pointing to a
         concrete csv file on Azure blob storage.
```

Comment:
This is not your change, but could you also update the links to `get_blob_to_text` and `create_blob_from_text` in the docstring, as they seem to be broken.

Reply:
Good spot! Fixed 🎉